### PR TITLE
Changes to pandas to match branding

### PR DESCRIPTION
**Pull request by matthewshaver**
## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Review by mirnawong1 — DISMISSED** [review]
approved @matthewshaver and have some small suggestions!

File: website/docs/docs/building-a-dbt-project/building-models/python-models.md [hunk 1]
```diff
@@ -522,7 +522,7 @@ def model(dbt, session):
 #### Code reuse
 
 Currently, Python functions defined in one dbt model cannot be imported and reused in other models. This is something we'd like dbt to support. There are two patterns we're considering:
```
Comment (mirnawong1) [inline website/docs/docs/building-a-dbt-project/building-models/python-models.md]:
```suggestion
Currently, you cannot import or reuse Python functions defined in one dbt model, in other models.  This is something we'd like dbt to support. There are two patterns we're considering:
```

File: website/docs/docs/building-a-dbt-project/building-models/python-models.md [hunk 2]
```diff
@@ -522,7 +522,7 @@ def model(dbt, session):
...
-1. Creating and registering **"named" UDFs**. This process is different across data platforms and has some performance limitations. (Snowpark does support ["vectorized" UDFs](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-batch.html): Pandas-like functions that can be executed in parallel.)
+1. Creating and registering **"named" UDFs**. This process is different across data platforms and has some performance limitations. (Snowpark does support ["vectorized" UDFs](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-batch.html): pandas-like functions that can be executed in parallel.)
```
Comment (mirnawong1) [inline website/docs/docs/building-a-dbt-project/building-models/python-models.md]: switch to active voice
```suggestion
1. Creating and registering **"named" UDFs**. This process is different across data platforms and has some performance limitations. (Snowpark does support ["vectorized" UDFs](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-batch.html): pandas-like functions that you can execute in parallel.)
```

File: website/docs/docs/building-a-dbt-project/building-models/python-models.md (hunk 1 above)
Comment (matthewshaver) [reply website/docs/docs/building-a-dbt-project/building-models/python-models.md]: Good catch, I missed this previously.  Thank you!

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
hey @matthewshaver merging this for you since it's been approved for a few months already. hope that's ok!

**Pull request merged**